headless-cms/package.json

```diff
@@ -24,6 +24,7 @@
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "react-refresh": "^0.8.3",
+    "sass": "^1.49.0",
```

Comment:
I would maybe remove this dependency and write it in pure CSS

Reply:
OK, I'll remove sass dependency.